### DISCUSSION TITLE
YOURLS Import Export Issues

### DISCUSSION THREAD

**Discussion by jamiers99** [YOURLS/YOURLS#3430] [Plugins]
Hi All,

I am working for a non-profit who is working to get disaster responders down to Florida for Hurricane Ian recovery.  I have setup a ton of short links into my YOURLS.  I would like to export them out and thought that using the [YOURLS Import Export Plugin](https://github.com/GautamGupta/YOURLS-Import-Export) would by my ticket to getting a list output in a CSV file.  

Unfortunately, it looks like this Plugin has been orphaned and it looks like it broke either with a PHP upgrade or a upgrade of YOURLS (reading from the issues tab).

I really need to get the links out so I can create a published list of my YOURLS installation so that I can get the list to our volunteer responders.

I would love to have the following items in some sort of non-pretty text list (would love to open it up in Excel as a text document even better), if someone knows of another plugin that might work
* Title
* Full Short Link
* Full Original URL

Thanks for any ideas...I just don't have the know how to develop it myself.

Thanks.

**Comment by ozh** [comment]
Ping @GautamGupta

Other options : export with a MySQL tool such as phpmyadmin, or hack your way from https://docs.yourls.org/development/examples/rss.html

**Reply by jamiers99** [reply]
I don't know why I didn't look there to begin with -- it gave me the list I needed.  Thanks @ozh

**Comment by gautamgupta** [comment]
Hi @jamiers99 I merged [this PR](https://github.com/GautamGupta/YOURLS-Import-Export/pull/11). Can you please re-download the [plugin](https://github.com/GautamGupta/YOURLS-Import-Export) and test whether the changes work for you?

If you still need help I can get on a call with you to debug it together — please email me directly at `g [at] gaut [dot] am`.
